app/views/devise/shared/_links.html.haml

```diff
@@ -5,7 +5,7 @@
   = link_to "Sign up", new_registration_path(resource_name), remote: true, class: 'shared_devise_link',  onclick: 'sharedDeviseLinkHandler(event);'
   %br/
 - if devise_mapping.recoverable? && controller_name != 'passwords' && controller_name != 'registrations'
-  = link_to "Forgot your password?", new_password_path(resource_name), class: 'shared_devise_link'
+  = link_to "Forgot your password?", new_password_path(resource_name)
```

Comment:
if you are not using the shared_devise_link class here, then need to check why we change it at all in the tailwind.scss file